content/en/about/community/customers/index.md

```diff
@@ -34,6 +34,7 @@ Tons of people are using Istio. Maybe you should too?
     {{< company_logo link="https://www.tuya.com" logo="./tuya.png" alt="Tuya Inc" >}}
     {{< company_logo link="https://www.ibm.com/watson" logo="./watson.png" alt="IBM Watson" >}}
     {{< company_logo link="https://ygrene.com" logo="./ygrene.jpeg" alt="Ygrene Energy Fund" >}}
+    {{< company_logo link="https://iherb.com" logo="./iherb.jpeg" alt="iHerb" >}}
```

Comment:
Entries are in alphabetical order, this one needs to be moved up. Also, the image is not working, see preview: https://deploy-preview-7597--preliminary-istio.netlify.app/about/community/customers/

Reply:
Image is failing as the link is specifying .jpeg, but the added file is .svg